docs/settings/apm-settings.asciidoc

```diff
@@ -9,7 +9,25 @@ You do not need to configure any settings to use APM. It is enabled by default.
 
 [float]
 [[general-apm-settings-kb]]
-==== General APM Settings
+==== APM x-pack Settings
```

Comment:
Fly-by comment: I recommend either not mentioning "X-Pack", or using {xpack} since we'll be removing that term as much as possible from the documentation in the near future.

Reply:
That's a good point. We have config options for two plugins: The x-pack APM plugin, and a meta OSS plugin, which is created for technical reasons. We might not need to discern between the two in the docs, but the prefix of the two are different (`xpack.apm.` vs `apm.`).

Reply:
@lcawl fyi I updated the copy